openapi/mojaloop-connector.yaml

```diff
@@ -0,0 +1,281 @@
+          required: false
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/mjl-transfer-rsp'
+        '500':
+          description: Invalid input
+components:
+  schemas:
+    mjl-quote-req:
+      required:
+        - transactionId
+        - quoteId
+        - payer
+        - payee
+        - amountType
+      type: object
+      properties:
+        transactionId:
+          type: string
+          example: '1067059f5c-e656-41ee-a746-631931d1ac12'
+        quoteId:
+          type: string
+          example: '1067059f5c-e656-41ee-a746-631931d1ac12'
+        extensionList:
+          type: array
+          items:
+            $ref: '#/components/schemas/mjl-party-info'
+          example: []
+        payer:
+          $ref: '#/components/schemas/mjl-party-info'
+        payee:
+          $ref: '#/components/schemas/mjl-party-info'
+        amountType:
+          type: string
+          description: The transaction type initiator type
+          enum:
+            - RECEIVE
+            - SEND
+        amount:
+          $ref: '#/components/schemas/mjl-amount'
+        transactionType:
+          $ref: '#/components/schemas/mjl-transaction-type'
+    mjl-quote-rsp:
+      required:
+        - transferAmount
+        - expiration
+      type: object
+      properties:
+        transferAmount:
+          $ref: '#/components/schemas/mjl-amount'
+        expiration:
+          type: string
+          example: '2022-12-06T09:47:12.783Z'
+        ilpPacket:
+          type: string
+          example: 'AYICFwAAAAAAAABkFGcudW5kZWZpbmVkLm1zaXNkbi4xggH2ZXlKMGNtRnVjMkZqZEdsdmJrbGtJam9pTjJZMVpEazNPRFF0TTJFMU55MDFPRFkxTFRsaFlUQXROMlJrWlRjM09URTFORGhoSWl3aWNYVnZkR1ZKWkNJNklqSXlORE5tWkdKbExUVmtaV0V0TTJGaVpDMWhNakV3TFRNM09EQmxOMll5WmpGbU5DSXNJbkJoZVdWbElqcDdJbkJoY25SNVNXUkpibVp2SWpwN0luQmhjblI1U1dSVWVYQmxJam9pVFZOSlUwUk9JaXdpY0dGeWRIbEpaR1Z1ZEdsbWFXVnlJam9pTVNKOWZTd2ljR0Y1WlhJaU9uc2ljR0Z5ZEhsSlpFbHVabThpT25zaWNHRnlkSGxKWkZSNWNHVWlPaUpOVTBsVFJFNGlMQ0p3WVhKMGVVbGtaVzUwYVdacFpYSWlPaUl4SW4xOUxDSmhiVzkxYm5RaU9uc2lZM1Z5Y21WdVkza2lPaUpGVlZJaUxDSmhiVzkxYm5RaU9pSXhJbjBzSW5SeVlXNXpZV04wYVc5dVZIbHdaU0k2ZXlKelkyVnVZWEpwYnlJNklrUkZVRTlUU1ZRaUxDSnBibWwwYVdGMGIzSWlPaUpRUVZsRlVpSXNJbWx1YVhScFlYUnZjbFI1Y0dVaU9pSkNWVk5KVGtWVFV5SjlmUQA'
+        condition:
+          type: string
+          example: 'xmHnYE0iQnMvi1CshISk9iYCf7MG3_ZsMNN9I4HKnAo'
+        payeeFspCommission:
+          $ref: '#/components/schemas/mjl-amount'
+        geoCode:
+          $ref: '#/components/schemas/mjl-geo-code'
+        payeeReceiveAmount:
+          $ref: '#/components/schemas/mjl-amount'
+        payeeFspFee:
+          $ref: '#/components/schemas/mjl-amount'
+    mjl-transfer-req:
+      required:
+        - transactionId
+        - payer
+        - payee
+        - amount
```

Comment:
the amount is just the same as the `quote request.amount`? correct?

Reply:
Yes. I don't think we have a use case where this will change.